index.js

```diff
@@ -1,42 +1,58 @@
 const Discord = require('discord.js');
+const {Uni} = require('./uniBot');
 const {prefix, token, giphyToken, ytToken} = require('./config');
 const client = new Discord.Client();
-const quotes = require('./quotes');
+
 const GphApiClient = require('giphy-js-sdk-core');
 giphy = GphApiClient(giphyToken);
 const fetch = require('node-fetch');
 
-client.once('ready', () => {console.log("Unicorn trotting!")});
+
+client.once('ready', () => {
+    console.log("Unicorn trotting!")
+});
 
 //!uni
 client.on('message', message => {
-    if (message.content.startsWith(`${prefix}uni`)) {
-        let member = message.mentions.members.first() || "";
-        giphy.search('gifs', {"q": ["magical", "unicorn", "animated candies", "fairyland"]})
-            .then((response) => {
-                //getting one gif
-                let totalResponses = response.data.length;
-                let responseIndex = Math.floor((Math.random() * 10) + 1) % totalResponses;
-                let finalResponse = response.data[responseIndex];
-                //getting one quote
-                let totalQuotes = quotes.length;
-                let quotesIndex = Math.floor((Math.random() * 10) + 1) % totalQuotes;
-                let quote = quotes[quotesIndex];
-
-                message.channel.send(`${quote} ${member} :unicorn:`, {
-                    files: [finalResponse.images.fixed_height.url]
-                });
-            })
-            .catch(() => {
-                message.channel.send('Hiccup...')
-            });
+    try {
+        if (message.content.startsWith(`${prefix}uni`)) Uni(message);
+        else if (message.content === 'ping') message.reply('pong');
+        else if (message.content === '!flip' || message.content === '!f') message.reply(Math.round(Math.random()) === 1 ? ' ♕ Heads!' : ' ♘ Tails!');
+        else if (message.content.startsWith('!scrim')) scrim(message)
+    } catch {
+        //message.reply('Something broke, should have written tests.');
     }
 });
 
-//ping-pong
-client.on('message', msg => {
-    if (msg.content === 'ping') msg.reply('pong');
-});
+const scrim = async (message) => {
+    const CHANNEL_CHEESE_CAKE = '310195830290382848';
```

Comment:
this id should probably not be in here. Maybe you can take the id of the channel to which the user is currently connected to. This way regardless of the channel, it will always work.

Reply:
something like `user.channelConnectedTo.id` ? Don't know exact notation, but I am sure something like this exists.

Reply:
yeah i like that

Reply:
that wont get into this iteration, if you wanna work on it the channel id could be obtained from the message im almost sure